docs/src/pages/[platform]/theming/responsive/examples/UseBreakpointValueObjectExample.tsx

```diff
@@ -1,12 +1,16 @@
-import { Alert, useBreakpointValue } from '@aws-amplify/ui-react';
+import {
+  Alert,
+  AlertVariations,
+  useBreakpointValue,
+} from '@aws-amplify/ui-react';
 
 export const UseBreakpointValueObjectExample = () => {
   const variation = useBreakpointValue({
     base: 'info',
     small: 'warning',
     medium: 'error',
     large: 'success',
-  });
+  }) as AlertVariations;
```

Comment:
I think you could `useBreakpointValue<AlertVariations>(` ?

Reply:
No, I cannot, useBreakpointValue is declared as a const and not a function, as such I cannot pass a generic parameter to it.

Reply:
That's my bad we should fix that.

Reply:
Let's add this to the list of docs type bugs to fix

Reply:
Added to backlog ticket.